src/app/pages/BridgeWithdrawPage/components/ConfirmStep/index.tsx

```diff
@@ -0,0 +1,216 @@
+/**
+ *
+ * BridgeDepositPage
+ *
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { Chain } from 'types';
+import { selectBridgeWithdrawPage } from '../../selectors';
+import { useWalletContext } from '@sovryn/react-wallet';
+import { toNumberFormat } from '../../../../../utils/display-text/format';
+import { Button } from '../../../../components/Button';
+import wMetamask from 'assets/wallets/metamask.svg';
+import { LinkToExplorer } from 'app/components/LinkToExplorer';
+import styled from 'styled-components/macro';
+
+import { useTranslation } from 'react-i18next';
+import { translations } from 'locales/i18n';
+
+import iconSuccess from 'assets/images/icon-success.svg';
+import iconRejected from 'assets/images/icon-rejected.svg';
+import iconPending from 'assets/images/icon-pending.svg';
+import { TxStep } from '../../../BridgeDepositPage/types';
+import { BridgeDictionary } from '../../../BridgeDepositPage/dictionaries/bridge-dictionary';
+import { NetworkModel } from '../../../BridgeDepositPage/types/network-model';
+import { CrossBridgeAsset } from '../../../BridgeDepositPage/types/cross-bridge-asset';
+import { AssetModel } from '../../../BridgeDepositPage/types/asset-model';
+import { SelectBox } from '../../../BridgeDepositPage/components/SelectBox';
+import { useHistory } from 'react-router-dom';
+
+interface Props {}
+
+export function ConfirmStep(props: Props) {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const { wallet } = useWalletContext();
+  const { amount, chain, targetChain, sourceAsset, tx } = useSelector(
+    selectBridgeWithdrawPage,
+  );
+  const currentNetwork = useMemo(
+    () =>
+      BridgeDictionary.listNetworks().find(
+        item => item.chain === chain,
+      ) as NetworkModel,
+    [chain],
+  );
+  const network = useMemo(
+    () =>
+      BridgeDictionary.listNetworks().find(
+        item => item.chain === targetChain,
+      ) as NetworkModel,
+    [targetChain],
+  );
+  const asset = useMemo(
+    () =>
+      BridgeDictionary.get(chain as Chain, targetChain as Chain)?.getAsset(
+        sourceAsset as CrossBridgeAsset,
+      ) as AssetModel,
+    [chain, sourceAsset, targetChain],
+  );
+  const handleComplete = useCallback(() => {
+    history.push('/wallet');
+  }, [history]);
+  return (
```

Comment:
Similar as in `BridgeDepositPage`, refactor `render` into more functions and use localized strings.